githubmarkdownui/blocks/leaf.py

```diff
@@ -25,4 +25,7 @@ def heading(text: str, level: int) -> str:
 
     :raises: Exception if the level is not between 1 to 6 inclusive
     """
-    pass
+    if not 1 <= level <= 6:
+        raise Exception('Level must be between 1 to 6 inclusive')
```

Comment:
I believe this is slightly more grammatically sound? I could be wrong though.
```suggestion
        raise Exception('Level must be between 1 and 6 inclusive')
```

Reply:
I think you're right, I've changed the docstring as well